examples/complete/stacks/orgs/cp/_defaults.yaml

```diff
@@ -33,3 +33,36 @@ helmfile:
 components:
   terraform: {}
   helmfile: {}
+
+settings:
+  atlantis:
+    # Select a config template defined in `atmos.yaml` in the `integrations.atlantis.config_templates` section
+    config_template_name: "config-1"
+
+    # Select a project template defined in `atmos.yaml` in the `integrations.atlantis.project_templates` section
+    project_template_name: "project-1"
+
+    # Workflow templates
+    # https://www.runatlantis.io/docs/custom-workflows.html#custom-init-plan-apply-commands
+    # https://www.runatlantis.io/docs/custom-workflows.html#custom-run-command
+    workflow_templates:
+      workflow-1:
+        plan:
```

Comment:
Does plan support the env key for arbitrary environment variables?

https://www.runatlantis.io/docs/custom-workflows.html#step

Reply:
Oh ok so something like this would work then since atlantis supports? That means that this object is now an any?

```yaml
    workflow_templates:
      workflow-1:
        plan:
          steps:
          - env:
              name: ENV_NAME
              value: hard-coded-value
          - env:
              name: ENV_NAME_2
              command: 'echo "dynamic-value-$(date)"'
```

If so, perfect :)